github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -15,15 +15,17 @@ postsubmits:
         nodeSelector:
           type: vm
         containers:
-          - image: kubevirtci/bootstrap:v20201119-a5880e0
+          - image: quay.io/kubevirtci/bootstrap:v20210302-4c35b4e
             command:
               - "/usr/local/bin/runner.sh"
               - "/bin/bash"
               - "-c"
               - >-
                 cat $DOCKER_PASSWORD | docker login --username $(<$DOCKER_USER)
                 --password-stdin && cd images &&
-                ./publish_image.sh rehearse docker.io kubevirtci
+                source ./publish_image.sh
```

Comment:
I'm not sure here, but I think this will not work as the main function will be called?

On my machine I tried the following:
```
❯ (source $GH/kubevirt.io/project-infra/images/publish_image.sh)                                                                                                                        
main:18: 1: parameter not set
```

Reply:
You are totally right, I've pushed changes to move the logic for getting the tag to a separate script and use it with `--workspace_status_command`, working locally, PTAL.

Reply:
Successful jobs for test https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/1016/pull-project-infra-test-rehearse/1369615885072863232 and publish image https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/logs/publish-rehearse-image/1369609468270088192